src/App.scss

```diff
@@ -15,25 +15,50 @@ button {
   color: red;
 }
 
+.App {
+  height: 100vh;
+  padding-top: 15px;
+  background: rgb(36,47,61);
```

Comment:
Consider using a more descriptive class name than `.App`. For example, `.AppContainer` or `.AppWrapper`. 📝